src/main/java/nl/tudelft/pixelperfect/Constants.java

```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.pixelperfect;
+
+/**
+ * A data class for storing game-wide items that can all be altered in a single sweep.
+ * 
+ * @author David Alderliesten
+ *
+ */
+public final class Constants {
+  // Gui-styling related constants for the in-game HUD.
+  public static final int guiWidthOffset = 150;
+  public static final int guiHeightOffset = 50;
```

Comment:
I do get checkstyle warnings about it, copied from maven site:

 Warning	naming	ConstantName	Name 'guiWidthOffset' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.	11
 Warning	naming	ConstantName	Name 'guiHeightOffset' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.	12
 Warning	naming	ConstantName	Name 'noEventsLogText' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.	15